openllm-core/src/openllm_core/config/configuration_baichuan.py

```diff
@@ -24,7 +25,8 @@
 \b
 $ openllm start baichuan --model-id='fireballoon/baichuan-vicuna-chinese-7b'
 '''
-DEFAULT_PROMPT_TEMPLATE = '''{instruction}'''
+DEFAULT_SYSTEM_MESSAGE = "A chat between a curious user and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the user's questions."
```

Comment:
What is the default system   message for baichuan english and mandarin?

might be better to also add an option for baichuan to change language. So we have default prompt for both Chinese and English.

Reply:
This is a good idea. But so far, I haven't been able to find official prompt templates for baichuan. I might be looking at the wrong places :) Would appreciate if you have any suggestions or links where I can take a look.

Reply:
You can take a look at their generate implementation.

Reply:
Hmm, it looks like the base models don't have a prompt_template. However, the chat variants have chat prompt templates (which we don't have to worry for now?). The default system message and prompt template I have now came from model-id `fireballoon/baichuan-vicuna-7b` and `hiyouga/baichuan-7b-sft`. If we want the DEFAULT_PROMPT_TEMPLATE to reflect the prompt template for the base model, then should it just be `'{instruction}'`?

Reply:
If we are using this as default system message, then we should also either change the default model id (The default model_id is probably not going to work with this prompt)

Reply:
After discussing with Larme, we believe that the default prompt template should be reverted back to '{instruction}'.